models/models.js

```diff
@@ -42,17 +47,44 @@ function fetchComments(article_id) {
     )
     .then((body) => {
       if (body.rows.length === 0) {
-        console.log(body.rows, "<-- rows in reject model");
         return Promise.reject({ status: 404, msg: "Not found" });
       } else {
         return body.rows;
       }
     });
 }
 
+function printComments(body, userName, article_id) {
```

Comment:
I would change the name of this model to be something more accurate like 'insertComment'